js/src/ui/MethodDecoding/methodDecoding.css

```diff
@@ -15,11 +15,23 @@
 /* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-.method {
-  font-family: 'Roboto Mono' !important;
+.details {
+}
+
+.description {
+  line-height: 1.5em;
+  padding-bottom: 0.5em;
+}
+
+.name,
+.inputs {
+  font-family: 'Roboto Mono';
```

Comment:
Should be `font-family: …, monospace;` to tell the computer that you want monospace event if loading the font fails.